migrations/1662032734061_create-compound-index-on-posts-tags-table.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable camelcase */
```

Comment:
@itzgeebee, maybe we're yet to get to this but just a reminder for us to take this out since we've clarified what the intention is

after that, we can merge it in.  Thanks

Reply:
Alright boss @jocrah

Reply:
@jakazzy

Reply:
sure thanks @itzgeebee